src/CentralPackageVersions/Sdk/Sdk.targets

```diff
@@ -133,6 +139,14 @@
     <PackageReference Include="@(_PackageReferenceWithVersionOverride)"
                       Condition=" '$(EnablePackageVersionOverride)' != 'false' "/>
 
+    <!--
+      Override version of <PackageReference /> items that declared in CentralPackagesFile if AllowPackageReferenceWithVersion
+      is set to true.
+    -->
+    <PackageReference Update="@(_PackageReferenceWithVersion)"
+                      Version="%(_PackageReferenceWithVersion.Version)"
```

Comment:
`_PackageReferenceWithVersion` is empty if `AllowPackageReferenceWithVersion` is `true`, so this won't actually do anything right?  Can you update the test to verify that the version is correct which will validate that this new logic is doing what you think it is?

Reply:
Gentle ping...

Reply:
Will update in recent days, being busy transferring team & rampup new project